### PR TITLE
[WIP] for diff

### PR DESCRIPTION
**Pull request by ForFishes**
It is used for diff. 
PR:
https://github.com/PaddlePaddle/Fleet/pull/91

### CODE REVIEW

File: examples/local_sgd/resnet/train.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
+ #copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
```
Comment (gavin1332) [inline examples/local_sgd/resnet/train.py]: unexpected blank

File: examples/local_sgd/resnet/train.py [hunk 2]
```diff
@@ -169,24 +193,21 @@ def get_device_num():
...
-        warm_up = [2*i for i in range(local_sgd_steps) if 2*i <=local_sgd_steps]
-        warm_up[0] = 1
-
-    warm_size = len(warm_up)
-    if offset >= warm_size:
-        return local_sgd_steps
-    else:
-        return warm_up[offset]
-     
+def AdaCom(ini_loss, ini_lr, cur_loss, cur_lr, base_step, pre_step):
```
Comment (gavin1332) [inline examples/local_sgd/resnet/train.py]: inconsistent format of function name

File: examples/local_sgd/resnet/train.py [hunk 3]
```diff
@@ -169,24 +193,21 @@ def get_device_num():
...
+    ratio = fir / sec
+    print("ratio {}".format(ratio))
+    step = int(base_step * math.sqrt(ratio))
+    print("ini_loss  {}  ini_lr  {}  cur_loss  {}  cur_lr  {}  base_step  {}  pre_step  {}  step  {}  sqrtstep  {}".format(ini_loss,
+         "%.5f"%ini_lr, "%.5f"%cur_loss, "%.5f"%cur_lr, base_step, pre_step, "%.5f"%step,"%.5f"% math.sqrt(ratio)))
+    if step < 1:
+        step = 1
+    if step > pre_step + 20:
+        step = pre_step
+    return int(step)
```
Comment (gavin1332) [inline examples/local_sgd/resnet/train.py]: do you intend to use floor?

File: examples/local_sgd/resnet/train.py (hunk 2 above)
Comment (gavin1332) [reply examples/local_sgd/resnet/train.py]: And you'd better put the reference paper for this algorithmn

File: examples/local_sgd/resnet/train.py (hunk 1 above)
Comment (ForFishes) [reply examples/local_sgd/resnet/train.py]: Done

File: examples/local_sgd/resnet/train.py (hunk 2 above)
Comment (ForFishes) [reply examples/local_sgd/resnet/train.py]: Done! add reference: https://arxiv.org/pdf/1810.08313.pdf

File: examples/local_sgd/resnet/train.py (hunk 3 above)
Comment (ForFishes) [reply examples/local_sgd/resnet/train.py]: Yes, rm the "int".

**Pull request closed without merging**